.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,54 +6,48 @@ labels: "bug"
 assignees: ""
 ---
 
-**Search terms you've used**
+<!-- 
+Thank you for reporting an issue.
 
-<!-- What search terms have you used to check whether this bug was already reported? -->
-
-**Describe the bug**
+Please fill in as much of the template below as you're able. Feel free to delete any section you don't think applies, but be aware that the more comprehensive your description, the easier it is to take it into account. 
+-->
 
-<!-- A clear and concise description of what the bug is. -->
+### Search terms you've used
+<!-- What search terms have you used to check whether this bug was already reported? -->
 
-**To Reproduce**
-Steps to reproduce the behavior:
+### Bug description
+<!-- A short description of what the problem is. -->
 
-1. Given this data: `…`
-2. Call this function `…`
-3. See error
+### To Reproduce
+1. 
+2. 
+3. 
+4. 
 
-**CodeSandbox**
+**Minimal reproduction**
 
 <!--
 Adding a minimal reproduction of the bug makes it as easy as possible to get it fixed.
-To do so, fork the following CodeSandbox, and share the resulting link here:
-
+To do so, fork the following CodeSandbox, insert the minimal code that demonstrates the problem,
+and share the resulting link here:
 https://codesandbox.io/s/github/inrupt/solid-client-js/tree/master/.codesandbox/sandbox
--->
-
-**Expected behavior**
 
-<!-- A clear and concise description of what you expected to happen. -->
+### Expected result
```

Comment:
Thanks. I did inject a syntax error in my suggestion, so here's the fix - sorry!

```suggestion
-->

### Expected result
```